keyboards/splitography/splitography.h

```diff
@@ -0,0 +1,38 @@
+/* Copyright 2017 Steven Dothum
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#ifndef SPLITOGRAPHY_H
+#define SPLITOGRAPHY_H
+
+#include "quantum.h"
+
+// This a shortcut to help you visually see your layout.
+// The first section contains all of the arguments
+// The second converts the arguments into a two-dimensional array
+
+#define KEYMAP( \
```

Comment:
You may want to format your matrix to something more like this: 
https://github.com/qmk/qmk_firmware/blob/master/keyboards/chimera_ortho/chimera_ortho.h#L66

For the top, you leave out the keys that aren't there... and use KC_NO on the bottom part to "zero out" the empty matrix locations.

That way, the keymaps only need to list the keys that are actually physically present. 

This isn't necessary, but it may make it more readable.